jenkins/user.sls

```diff
@@ -3,6 +3,7 @@
 jenkins-user:
   user.present:
     - name: jenkins
+    - uid: 1000
```

Comment:
Ubuntu policy is for software packages requiring a user to have their UID be < 1000.
http://people.canonical.com/~cjwatson/ubuntu-policy/policy.html/ch-opersys.html.  >=1000 is for actual human accounts. Ideally we should have it in this lower range.

Ideally also set
```
- system: True
```

such that this property is preserved even if the numerical reservation is later dropped.